### ISSUE TITLE
Add Network information 

### ISSUE THREAD

**Issue by lfreist** [lfreist/hwinfo#16]
- [ ] Interface 
- [ ] MAC
- [ ] IP

**Comment by MRsoymilk** [comment]
Hello, I did network part of Windows (inlcude interface index, MAC, IPv4), but there is something wrong with the pull request, can you provide some help, thanks

**Comment by lfreist** [comment]
Sure, can you provide me some more information about the issue?

**Comment by MRsoymilk** [comment]
I already create a draft pull request, can you see it in the pull requests? you can directly edit it, really thanks.

**Comment by lfreist** [comment]
Yes, I can see it. I will review it in the next days as soon as I have some time left! Thank you for the contribution.
